library/src/main/java/org/xmtp/android/library/Conversations.kt

```diff
@@ -388,12 +392,14 @@ data class Conversations(
                             val decoded = conversation.decode(envelope)
                             emit(decoded)
                             topics.add(conversation.topic)
-                            currentCoroutineContext().job.cancel()
+                            subscribeFlow.value = makeSubscribeRequest(topics)
                         }
 
                         else -> {}
                     }
                 }
+            } catch (error: CancellationException) {
+                break
```

Comment:
Only break if it's a cancellation request for the stream? Or should we break for all exceptions?